src/components/Icon/index.js

```diff
@@ -40,6 +41,9 @@ const Icon = ({ name, size, className, src, button, children, style, sizePx, ...
       {...props}
       className={cx(styles.container, styles[size], className, button && styles.button)}
       style={buildStyle({ style, src, name, imageSrc, sizePx })}
+      role="button"
+      tabIndex="0"
```

Comment:
is tab index needed?

Reply:
It is for a `<div>` in the higher versions of eslint to avoid the error. It shouldn't be when I switch to `<a>`.